charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0.0"
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 1.2.0
+version: 1.3.0
```

Comment:
correct me if I'm wrong.. it should be 1.0.3 instead of 1.3.0 as the driver version is still on 1.0.x

Reply:
I think, if we avoid exposing env variable, then we dont need to change the chart at all

Reply:
the helm chart version is 1.2.0 on master right now. is that a problem?

Reply:
In general, the helm chart version doesn't have to and likely won't match the version of the application.  That is what appVersion is for to specify the version of the application to use if the user doesn't override it.  If the chart keeps getting new features added that work with 1.0.0. of the driver, it will continue to increase in version independent of the driver version.